server/phpunit.xml.dist

```diff
@@ -26,11 +26,14 @@
     <testsuites>
         <testsuite name="Domain">
             <directory>tests/Unit/Administration/Domain</directory>
+            <directory>tests/Unit/Company/Domain</directory>
             <directory>tests/Unit/Core/Domain</directory>
+            <directory>tests/Unit/User/Domain</directory>
         </testsuite>
-
         <testsuite name="End2End">
             <directory>tests/End2End/Administration/Infrastructure</directory>
+            <directory>tests/End2End/Company/UI</directory>
+            <directory>tests/End2End/User/UI</directory>
```

Comment:
```suggestion
            <directory>tests/End2End</directory>
```

Reply:
Why not only `<directory>tests/End2End</directory>` ?

Reply:
In order to simplify and avoid always having to add a new line when creating a domain